public/app/features/dashboard/state/__fixtures__/dashboardFixtures.ts

```diff
@@ -0,0 +1,35 @@
+import { Dashboard, defaultDashboardCursorSync, Panel } from '@grafana/schema';
+import { DashboardMeta } from 'app/types';
+
+import { DashboardModel } from '../DashboardModel';
+
+export function createDashboardModelFixture(
```

Comment:
Can this be a factory function (static newXxx or createXxx) on DashboardModel instead? Would be more discoverable

Reply:
I don't have any confidence around these values being correct/make sense for runtime. Do you think there's much risk from this there? Do we _want_ these current defaults to be discoverable in for people working on runtime/non-tests?

Reply:
Do you have thoughts about how a previous version of this looked? https://github.com/grafana/grafana/pull/57896/commits/803f1c2cf56d28346c355aa689fac3cf63c894cf#diff-4b35669fc9cf4d73ef70100b81f61f392b9454aff3234f3964fc6cfef36fb87fR3